charts/k8ssandra/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: k8ssandra
 description: |
   Provisions and configures an instance of the entire K8ssandra stack. This includes Apache Cassandra, Stargate, Reaper, Medusa, Prometheus, and Grafana.
 type: application
-version: 0.60.0
+version: 0.60.1
```

Comment:
I don't know why GH keeps failing to show the right diffs for the chart versions, same thing happened the other day.  In any case, we'll need to bump here, 60.1 is already current:

https://github.com/k8ssandra/k8ssandra/blob/ecdc4f1d78422b91c6ab4045e776314286cf8834/charts/k8ssandra/Chart.yaml#L6